src/cmd/apply.ts

```diff
@@ -109,7 +115,7 @@ const applyAll = async () => {
   await upgrade({ when: 'post' })
   if (!(env.isDev && env.DISABLE_SYNC)) {
     await commit()
-    if (intitalInstall) {
+    if (initialInstall && !argv.tekton) {
```

Comment:
Could you use `!argv.tekton` while defining the value of the initialInstall variable?